### DISCUSSION TITLE
Extract no @gmail email

### DISCUSSION THREAD

**Discussion by Lalladub** [dbeaver/dbeaver#34058] [Q&A] [answered]
Dear team,
I have a request from the client:
I need to extract from DBeaver all emails with no @gmail acquired between 18 and 24 months ago.

Is it possibile? If yes, can you please suggest a query?

Many thanks

Lavinia

**Comment by E1izabeth** [comment]
Hello, @Lalladub 
We're not familiar with your data structure, so we can't help. The only thing I can say is that it's surely possible by writing and executing SQL query.

**Comment by Lalladub** [comment, marked as answer]
Thanks Elisabeth,
I'm very new to SQL language so I was in doubt but it worked with: 

SELECT email
FROM your_table_name
WHERE email NOT LIKE '%@gmail.com'

thanks
